app/uk/gov/hmrc/gform/core/parsers/BooleanExprParser.scala

```diff
@@ -60,6 +60,9 @@ object BooleanExprParser {
     | exprFormCtx ~ ">" ~ exprFormCtx ^^ { (loc, expr1, op, expr2) =>
       GreaterThan(expr1, expr2)
     }
+    | exprFormCtx ~ "includes" ~ exprFormCtx ^^ { (_, expr1, _, expr2) =>
```

Comment:
We can change this to
```
| alphabeticOnly ~ "includes" ~ exprFormCtx ^^ { (_, expr1, _, expr2) =>
      Includes(FormCtx(expr1), expr2)
```
where logic later will verify that whatever is captured by `alphabeticOnly` is valid `FormComponentId`.